airflow/contrib/operators/databricks_operator.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+import time
+
+from airflow.exceptions import AirflowException
+from airflow.contrib.hooks.databricks_hook import DatabricksHook
+from airflow.models import BaseOperator
+
+POLL_SLEEP_PERIOD_SECONDS = 5
+LINE_BREAK = ("-" * 80)
+
+
+class DatabricksSubmitRunOperator(BaseOperator):
+    """
+    Submits an ephemeral run to Databricks and waits for it to complete
+    successfully.
+
+    For more information about Databricks ephemeral runs look at
+    https://docs.databricks.com/api/latest/jobs.html#runs-submit
+    """
+    def __init__(self,
```

Comment:
we need document at least some parameters in python - for example, extra_api_parameters. I would call it extra_data though:0